### PR TITLE
Pull Request for Apple Clone Project 

### PR DESCRIPTION
**Pull request by karthi07**
test

### CODE REVIEW

**Comment by karthi07** [comment]
Worked on following functionalities

Clone the Apple Web page
learnt and implemented z index
Multiple layout options
setting background image and positioning element according to it

PS : fixed lint error. after setting up github stickler plugin, the checks was mentioned in the comments [fixed all the lint errors on checking from comments] . let me know if it's possible to add to check.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey @karthi07 , 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯

- Good Readme file
- Good Pull Request Message

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Always remember to check your HTML using the [validator](https://validator.w3.org/). You have some errors and warnings. Please fix them.
Screenshot from 2020-03-12 17-32-42

## Please, make all REQUIRED changes (check below, too) and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                <li><a href="#"> Mac</a></li>
+                <li><a href="#"> iPod</a></li>
+                <li><a href="#"> iPhone</a></li>
+                <li><a href="#"> iPad</a></li>
+                <li><a href="#"> iTunes</a></li>
+                <li><a href="#"> Support</a></li>
+                <li><a href="#"> 
+                        <div class="nav-search">
+                        <input type="text" name="search" id="search-bar">
+                        <div class="sr-img"></div>
```
Comment (majovanilla) [inline index.html]: - Your search icon is not showing. Please make it appear inside the search bar.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            </li>
+            <li>
+                <a href="#">
+                    <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg" alt="Supplier">
+                </a>
+            </li>
+            </ul>
+        </aside> 
+
+        <footer>
```
Comment (majovanilla) [inline index.html:82]: - Your footer should be centered. Please fix it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            <div class="footer-links">
+            <p>Shop the <a href="#"> Apple Online Store </a> (1-800-MY-APPLE), visit an <a href="#"> Apple Retail Store </a>, or find a <a href="#"> reseller</a>.</p>
+            
+            <ul class="right-links">
+                <a href="#">
```
Comment (majovanilla) [inline index.html]: Each `<li>` element should be an individual `<a>` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                <li>Job Opportunities</li>
+                <li>Contact Us</li>
+                <li><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/global/elements/flags/16x16/usa.png" alt=""></li>
+            </a>
+            </ul>
+
+            <div class="footer-right">
+                <p>Copyright © 2014 Apple Inc. All rights reserved.</p>
+                <ul>
+                    <a href="#">
```
Comment (majovanilla) [inline index.html]: Each `<li>` element should be an individual `<a>` tag

**Review by majovanilla — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (karthi07) [reply index.html:82]: Screenshot from 2020-03-13 09-43-15
Footer is centered in my screen width settings, if not please share your width setting

**Comment by karthi07** [comment]
@majovanilla  Completed all the changes :) Regarding the footer width center, it works on my current screen size, let know which size need to be changed.

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @karthi07 :handshake: 

### Thank you for fixing the changes requested by the previous reviewer :medal_sports: and good job taking care of all browsers when thinking in your navbar gradient :star: 

### Now we have to take care of your page looking good in all screens :point_down: 

## % for width

- [ ] I see you hardcoded values for width, try using percentages and make the heroe section background and aside content take 100% of the screen width, so it will look proportional in all screen sizes, in my screen is not only the footer that looks out of place is also the navbar almost overflows the heroe background

your page in my screen:

image

original in my screen:

image

### That's it! :raised_hands: 

### Please :pray: fix this :man_technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any questions? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by karthi07** [comment]
Screenshot from 2020-03-13 12-25-18
Converted the hardcorded width to percentages for body (hero) and aside. Nav and footer's widht are in percentages. those sections works perfectly on inspect element responsive tab too. the screen settings are based on the apple page and background image. Tried all my possible options i know, let me know if i can change something here

**Comment by lekegitrepo** [comment]
## Hi @karthi07 💯 
### You have done a great job by making the required changes and also for making the page look almost like the original page. This project is approved 🥇 

### Consider these suggestions as optional:
- You can try to add the hover effect in the navbar like the original.
- Make the apple logo a link (i.e. you can wrap it within a link tag.
- You should write a descriptive pull request comment instead of the word test that you put in your pull request comment.

### Reviewed by: Temitayo Adeleke.

-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

**Review by lekegitrepo — APPROVED** [review]
### Status: Approved

**Pull request merged**